fortio_main.go

```diff
@@ -103,8 +103,8 @@ var (
 
 	allowInitialErrorsFlag = flag.Bool("allow-initial-errors", false, "Allow and don't abort on initial warmup errors")
 	autoSaveFlag           = flag.Bool("a", false, "Automatically save JSON result with filename based on labels & timestamp")
-	redirectFlag           = flag.Int("redirect-port", 8081,
-		"Redirect all incoming traffic to https URL (need ingress to work properly). -1 means off.")
+	redirectFlag           = flag.String("redirect-port", "disabled",
+		"IP:Port pair to redirect all incoming traffic to https. Disabled by default.")
```

Comment:
why change the default ?

Reply:
I'll revert. IMO using "disabled" is more intuitive for a user, but it's outside the scope of the listen addr issue this PR intends to resolve.

Reply:
once we move it as a string yes, but I was specifically asking about why change the default from listening on 8081 to not listening